tests/common.go

```diff
@@ -5221,3 +5221,34 @@ func GetRandomStorageLessNode(slNodes []node.Node) node.Node {
 	}
 	return node.Node{}
 }
+
+// GetPoolIDsFromVolName returns list of pool IDs associated with a given volume name
+func GetPoolIDsFromVolName(volName string) ([]string, error) {
+	/* This method will pick all the pools associcated with the volume
+	and return the UUID of the pools in array format
+	*/
```

Comment:
There is no need for this comment, you already explained what this function does above

Reply:
sure thanks !